tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventAcceptMethodContract.scala

```diff
@@ -221,8 +215,8 @@ trait LinagoraCalendarEventAcceptMethodContract {
   }
 
   @Test
-  def acceptAMissingAttendeeIcsShouldReturnAccepted(server: GuiceJamesServer): Unit = {
-    setupServer(server, DEFAULT_INVITATION_DATA)
+  def acceptAMissingAttendeeIcsShouldReturnAccepted(server: GuiceJamesServer, eventInvitation: EventInvitation): Unit = {
+    setupServer(server, eventInvitation)
```

Comment:
This test case is incorrect. Sending an ICS without attendees results in a `400 BAD_REQUEST` from the Dav server. A better approach is to introduce custom validations that return notAccepted if the ICS has no attendees or the current user isn’t listed. I’m disabling this for now to maintain a small PR.

Reply:
Need to be checked